project_tests.sh

```diff
@@ -16,7 +16,7 @@ proofreader app/ bin/ src/ web/
 proofreader --no-phpcpd features/ test/
 
 echo "PHPUnit tests"
-vendor/bin/phpunit --log-junit build/phpunit.xml
+vendor/bin/paratest test --log-junit build/phpunit.xml --runner WrapperRunner
```

Comment:
This project seems to have evolved well despite the lack of support from the official PHPUnit (which was due to implement parallelization 1 or 2 major versions ago)